changelogs/fragments/948-dellemc-migration-removal.yml

```diff
@@ -0,0 +1,13 @@
+removed_features:
+  - |
+    ``ome_device_info``, ``idrac_firmware`` and ``idrac_server_config_profile``  modules and plugins have now been migrated from community.general to the `dellemc.openmanage <https://galaxy.ansible.com/dellemc/openmanage>`_ Ansible collection.
+    If you use ansible-base 2.10 or newer, redirections have been provided.
+
+    If you use Ansible 2.9 and installed this collection, you need to adjust the FQCNs (``community.general.idrac_firmware`` → ``dellemc.openmanage.idrac_firmware``) and make sure to install the dellemc.openmanage collection.
+breaking_changes:
+  - |
+    If you use Ansible 2.9 and these plugins or modules from this collection, community.general 3.0.0 results in errors when trying to use the DellEMC content by FQCN, like ``community.general.idrac_firmware``.
+    Since Ansible 2.9 is not able to use redirections, you will have to adjust your playbooks and roles manually to use the new FQCNs (``dellemc.openmanage.idrac_firmware`` for the previous example) and to make sure that you have ``dellemc.openmanage`` installed.
+
+    If you use ansible-base 2.10 or newer and did not install Ansible 3.0.0, but installed (and/or upgraded) community.general manually, you need to make sure to also install the ``dellemc.openmanage`` collection if you are using any of these plugins or modules.
```

Comment:
```suggestion
    If you use ansible-base 2.10 or newer and did not install Ansible 4.0.0, but installed (and/or upgraded) community.general manually, you need to make sure to also install the ``dellemc.openmanage`` collection if you are using any of these plugins or modules.
```

Reply:
@felixfontein , are we not good to include the `dellemc.openmanage` in 3.0.0 release?

Reply:
I'm right now assuming that community.general 3.0.0 will be contained in Ansible 4.0.0 - that's not guaranteed yet, but it definitely won't be contained in Ansible 3.x.y. So it's better to change this to 4.0.0, so we can adjust it if necessary.

> I just want to reconfirm with you that our `dellemc.openmanage` collections are going to be included in Ansible 3.0.0 release coming out in Feb-16, right?

It's my understanding that this is correct.

Reply:
Thanks @felixfontein 

There are two places we need to replace 3.0.0  to 4.0.0 in the same paragraph, right? One the place you suggested and other one in the last line `While ansible-base 2.10 or newer can use the redirects that community.general 3.0.0 adds,`

I will update both to 4.0.0